i6_models/parts/conformer/mhsa.py

```diff
@@ -39,17 +39,21 @@ def __init__(self, cfg: ConformerMHSAV1Config):
         )
         self.dropout = cfg.dropout
 
-    def forward(self, input_tensor: torch.Tensor, key_padding_mask: Optional[torch.Tensor] = None) -> torch.Tensor:
+    def forward(self, input_tensor: torch.Tensor, sequence_mask: Optional[torch.Tensor] = None) -> torch.Tensor:
```

Comment:
Do we really want to allow to have it optional?

Reply:
We need this for export. During export, from our current knowledge it is almost impossible to generate a sequence mask (because all variants @JackTemaki and I found require `max_len` as int and/or torch.arange (which needs int input)). This is not exportable. One could write a custom op, but we decided against it for now because we don't do batch decoding anyway, this could be later added I guess if required. 
And then simply putting the mask to None in the initial Module is the solution we came up with.

Reply:
What code did you try? I just tried `torch.arange(x.shape[0])` for some tensor `x` and that seems to work fine?

Reply:
My testing code: https://github.com/albertz/playground/blob/master/torch-tracing-arange.py

Reply:
Hmm, you are right this seems to work. I have to admit most of the testing was done by @JackTemaki, but like this the export of the conformer works for me. 
I would wait for @JackTemaki s feedback here.

Reply:
I extended the demo code for doing a seq mask, and that also works just fine: https://github.com/albertz/playground/blob/master/torch-tracing-seqmask.py

Reply:
So it seems @albertz variant is working. This means I will remove the support for optional mask here.